test/decl/inherit/inherit.swift

```diff
@@ -46,7 +46,9 @@ struct S2 : struct { } // expected-error{{expected type}}
 struct S3 : P, P & Q { } // expected-error {{redundant conformance of 'S3' to protocol 'P'}}
                          // expected-error @-1 {{non-class type 'S3' cannot conform to class protocol 'Q'}}
                          // expected-note @-2 {{'S3' declares conformance to protocol 'P' here}}
-struct S4 : P, P { }     // expected-error {{duplicate inheritance from 'P'}}
+struct S4 : P, P { }     // FIXME: expected-error {{duplicate inheritance from 'P'}}
+// expected-error@-1{{redundant conformance of 'S4' to protocol 'P'}}
+// expected-note@-2{{'S4' declares conformance to protocol 'P' here}}
```

Comment:
I think the duplicate diagnostics come from `checkInheritanceClause()` and the GSB, respectively

Reply:
Yeah. I'm going to narrow the one in `checkInheritanceClause()` to eliminate the duplication. See https://github.com/apple/swift/pull/18700